src/components/Navbar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] wonderful job so far, app works and looks like expected. Please kindly note that disabling ESLint is not a good practice, as it helps enforce coding standards and catch potential errors. Please kindly fix the errors normally as this will not only improve code quality, make it easier to read and maintain, and reduce the risk of bugs but will also advance your skills. 
- Please kindly do same for the ones in calculator, home and app components. Nice work. 😄

Reply:
Fixed.